obal/data/roles/build_package/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- include_tasks: koji.yml
```

Comment:
`include_tasks: "{{ build_package_build_system }}.yml"`? that allows to drop the `when`

also please add a `name`

Reply:
Yup